content/tutorial-deep-reinforcement-learning-with-pong-from-pixels.ipynb

```diff
@@ -0,0 +1,1275 @@
+{
```

Comment:
<p>When I check this in reviewnb your list does not get rendered properly. Not sure if this is just reviewnb or a problem with the markdown (you may have to break a line before the list so that it gets correctly parsed)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
🤔 It looked ok in my reviewNB instance. To be safe, I added blank lines before the start of each "nested sub-list". Hopefully, it look better now!